docs/overview.md

```
@@ -0,0 +1,276 @@
+
+# Core CLI Scripts Overview
+We provide a set of core CLI scripts to facilitate the tabularization and modeling of MEDS data. These scripts are designed to be run in sequence to transform raw MEDS data into tabularized data and train a model on the tabularized data. The following is a high-level overview of the core CLI scripts:
+
+#### 1. **`MEDS_transform-reshard_to_split`**:
```

Markdownlint

Expected: h2; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)